docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -346,7 +346,7 @@ public class FruitCodec implements CollectibleCodec<Fruit> {
     private final Codec<Document> documentCodec;
 
     public FruitCodec() {
-        this.documentCodec = MongoClient.getDefaultCodecRegistry().get(Document.class);
+        this.documentCodec = MongoClientSettings.getDefaultCodecRegistry().get(Document.class);
```

Comment:
Do we need a PR for the quickstart too (if there is one)?

Reply:
Yes, I'll send it today or tomorrow.

As we add some stuff to Hibernate with Panache and MongoDB with Panache lately I will check these two quickstart also to align them with the guides.